docs/pages/enterprise/workflow/ssh-approval-pagerduty.mdx

```diff
@@ -72,6 +72,9 @@ and Teleport Auth Service access.
 Run `./install` from `teleport-pagerduty` or place the executable in the appropriate `/usr/bin` or `/usr/local/bin` on the server installation.
 </TabItem>
 <TabItem label="Docker">
+
+ Make a directory called `config` to store the access-plugin certificate and tls/https certs.
```

Comment:
This sentence looks a bit out of place here. Shouldn't it be in the "config file" section below rather than where Docker install instructions are? Or maybe just remove it since instructions below do `mkdir config` explicitly.